index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Simple Art</title>
+
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <div class="header__logo">
+          <a href="#">
+            <img src="./images/logo-SA-1.svg" alt="Simple Art logo" />
+          </a>
+        </div>
+        <div class="header__bar">
+          <nav class="header__navigation">
+            <ul class="header__navigation-list list">
+              <li class="header__navigation-list-item">
+                <a class="header__navigation-link" href="#">Courses</a>
+              </li>
+              <li class="header__navigation-list-item">
+                <a class="header__navigation-link" href="#">Teachers</a>
+              </li>
+              <li class="header__navigation-list-item">
+                <a class="header__navigation-link" href="#">Prices</a>
+              </li>
+              <li class="header__navigation-list-item">
+                <a class="header__navigation-link" href="#">About us</a>
+              </li>
+            </ul>
+          </nav>
+          <button class="btn btn--slim header__btn">Apply now</button>
```

Comment:
Don't forget to specify the `type `attribute for the `<button>` tag, as its default behavior is `submit`.